### PR TITLE
Bulk record creations.

### PR DESCRIPTION
**Pull request by n1k0**
Quickly hacked as a PoC.

Feedback=? @leplatrem 

Tests will follow.

### CODE REVIEW

**Comment by n1k0** [comment]
Deployed a demo to gh-pages: kinto.github.io/kinto-admin/#/collections/tasks/bulk

File: scripts/containers/BulkFormPage.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    name: state.collection.name,
+    schema: state.collection.schema,
+    uiSchema: state.collection.uiSchema,
+    form: state.form,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return bindActionCreators(
+    Object.assign({}, CollectionActions, NotificationsActions, {updatePath}),
```
Comment (leplatrem) [inline scripts/containers/BulkFormPage.js]: nit: rest spread plugin is available

Comment (n1k0) [inline scripts/containers/BulkFormPage.js]: yeah, then I have other files to update for consistency

File: scripts/components/BulkForm.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  onSubmit(data) {
+    this.props.bulkCreate(data.formData);
+  }
+
+  render() {
+    const {name, schema, uiSchema} = this.props;
+    const bulkSchema = {type: "array", items: schema};
+    const bulkUiSchema = {items: uiSchema};
```
Comment (leplatrem) [inline scripts/components/BulkForm.js]: nice! :)

**Comment by leplatrem** [comment]
r+ after having tried the demo :) works as expected!

**Pull request merged**